base/workflow/inst/batch_run.R

```diff
@@ -45,122 +45,119 @@ pecan_path <- get_arg(argv, "--pecandir", getwd())
+    php_file <- file.path(pecan_path, "web", "config.php")
+    stopifnot(file.exists(php_file))
+    config.list <- PEcAn.utils::read_web_config(php_file)
+    bety <- PEcAn.DB::betyConnect(php_file)
+    con <- bety$con
+    
+    # Get model ID
+    table_row <- input_table[i, ]
+    model <- table_row$model
+    revision <- table_row$revision
+    message("Model: ", shQuote(model))
+    message("Revision: ", shQuote(revision))
+    
+    
+    
+    model_df <- tbl(con, "models") %>%
+      filter(model_name == !!model,
+             revision == !!revision) %>%
+      collect()
+    
+    if (nrow(model_df) == 0) {
+      message("No models found with name ", model,
+              " and revision ", revision, ".\n",
+              "Moving on to next row.")
+      next
+    } else if (nrow(model_df) > 1) {
+      print(model_df)
+      message("Multiple models found with name ", model,
+              " and revision ", revision, ".\n",
+              "Moving on to next row.")
+      next
+    } else {
+      model_id <- model_df$id
+    }
+    
+    pft <- table_row$pft
+    if (is.na(pft)) pft <- NULL
+    
+    # Run test
+    raw_result <- create_execute_test_xml(
+      model_id = model_id,
+      met = table_row$met,
+      site_id = table_row$site_id,
+      pft = pft,
+      start_date = table_row$start_date,
+      end_date = table_row$end_date,
+      dbfiles_folder = dbfiles_folder,
+      pecan_path = pecan_path,
+      user_id = user_id,
+      ensemble_size = table_row$ensemble_size,
+      sensitivity = table_row$sensitivity,
+      output_folder=output_folder
+    )
+    
+  })
 
-  outdir <- raw_result$outdir
-  result_table$outdir <- outdir
 
-  ##################################################
-  # Did the workflow finish?
-  ##################################################
-  raw_output <- readLines(file.path(outdir, "workflow.Rout"))
-  result_table$workflow_complete[[i]] <- any(grepl(
-    "PEcAn Workflow Complete",
-    raw_output
-  ))
-  continue <- FALSE
-  ##################################################
-  # Did we write a job.sh file?
-  ##################################################
-  out <- file.path(outdir, "out")
-  run <- file.path(outdir, "run")
-  jobsh <- list.files(run, "job\\.sh", recursive = TRUE)
-  ## pft <- file.path(outdir, "pft")
-  if (length(jobsh) > 0) {
-    result_table$has_jobsh[[i]] <- TRUE
-    continue <- TRUE
-  }
 
-  ##################################################
-  # Did the model produce any output?
-  ##################################################
-  if (continue) {
-    continue <- FALSE
-    raw_out <- list.files(out, recursive = TRUE)
-    if (length(raw_out) > 0) {
-      result_table$model_output_raw[[i]] <- TRUE
-      continue <- TRUE
+#----------- Checking the results of the runs
+checks_df<-list.dirs(output_folder, full.names = TRUE, recursive = FALSE) %>% 
+  purrr::map_dfr(function(outdir){
+    
+    result_table <-NULL
```

Comment:
Per my comment below, the original idea here was that the result table would have all of the information for the run, plus the additional columns indicating run status. Why drop it here?